app/pages/ServiceDiscoveryResults/ServiceDiscoveryResults.scss

```
@@ -63,9 +68,13 @@
     margin-left: 0;
     cursor: pointer;
   }
+
+  @media screen and (max-width: 767px) {
```

```suggestion
  @media screen and (max-width: 600px) {
```

It's kind of strange to use this breakpoint. Why not use the same 600px breakpoint seen elsewhere on this page? Alternatively, you could make all of the 600px breakpoints be 767px, but it's just a bit confusing to have so many different breakpoints on the page at the same time.